src/components/Onboarding/Navigation.js

```diff
@@ -8,6 +8,7 @@ const buttonTransitionStyles = show => ({
   opacity: Number(show),
   transform: `translate3d(0, ${show ? 0 : 2}px, 0)`,
   config: springs.swift,
+  pointerEvents: show ? 'auto' : 'none',
```

Comment:
Another little fix here: The modal buttons have pointer-events even if they are not being shown as only the opacity changes, and this prevents this behavior. The upgrade organization modal from the client also had this, and during spring cleaning [we applied the same fix](https://github.com/aragon/aragon/blob/master/src/components/Upgrade/Navigation.js#L10).